books/management/commands/addepub.py

```diff
@@ -249,7 +249,7 @@ def process_epub(self, filename, use_symlink=False):
                                 .split(',')
                             for tag in subject_split:
                                 if tag is not ' ':
-                                    tag = tag.encode("utf-8").lower().strip()
+                                    tag = tag.lower().strip()
```

Comment:
It looks like the specs do specify encoding:

"OPF Package Documents may use the entire Unicode character set, in UTF-8 or UTF-16 encodings …"

http://www.idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section1.3.4

though from what I recall, some were in ascii. As you mentioned in another comment, I'll collect some sample output and then we can see for sure how well OPFs follow the spec.